packages/test-utils/src/testUtils/apis/AnalyticsApi/MockAnalyticsApi.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { AnalyticsApi, AnalyticsEvent } from '@backstage/core-plugin-api';
+
+export class MockAnalyticsApi implements AnalyticsApi {
+  private events: AnalyticsEvent[] = [];
```

Comment:
So ... is there a convenient way of making things like this be reset by `jest.reset/clearAllMocks()`? Kinda would be in line with expectations for a consumer! (don't worry about it too much, i just started thinking out loud)

Reply:
If so, we should also apply it to `MockErrorApi`, upon which this is based.  Although I think in actual usage, you end up just instantiating new mocks?